### ISSUE TITLE
Should MSR skip private packages

### ISSUE THREAD

**Issue by StevenLangbroek** [dhoulb/multi-semantic-release#34]
Hey folks! We have a package in `packages`, a "kitchen sink" for our design system, which shouldn't be published (we have it as a workspace package so that it can depend on other packages like our consumers would). I thought by setting `"private": true` in its package.json it wouldn't be published, but it was. Do you think this is the right behavior? Or should it skip private packages? (would gladly contribute if you think this behavior should change)

**Comment by dhoulb** [comment]
Yep private packages are supposed to be skipped, so not sure how that happened!

It's not something we mess with manually, they should be skipped by virtue of running the normal `semanticRelease()` API on the CWD of the workspace package (and then semantic release sees the `"private": true` and skips the release for that package). There's no real reason I can think of that it wouldn't work unless it's a bug in semantic release itself.

**Comment by StevenLangbroek** [comment]
Well, this is the package's package.json:
![image](https://user-images.githubusercontent.com/296796/94943593-3bf08e80-04d8-11eb-80f3-ce11612f2089.png)

But it releases every time... What can I do to help? Open an issue against `semantic-release`?

**Comment by dhoulb** [comment]
If you CD into the directory and run the `semantic-release` CLI against it (in test mode), does it release too?

**Comment by esatterwhite** [comment]
I think that behavior is define by [npm](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private)

**Comment by moroine** [comment]
I think https://github.com/dhoulb/multi-semantic-release/pull/76 fixes this issue
